pytest/lib/account.py

```diff
@@ -83,3 +83,10 @@ def send_stake_tx(self, stake_amount):
         tx = sign_staking_tx(self.key, self.key, stake_amount, self.nonce,
                              self.base_block_hash)
         return self.send_tx(tx)
+
+    def get_amount_yoctonear(self):
+        j = self.json_rpc('query', {
+            'request_type': 'view_account',
+            'finality': 'optimistic',
+            'account_id': self.key.account_id})
+        return int(j.get('result',{}).get('amount',0))
```

Comment:
That’s essentially `get_account` from `BaseNode`.

Reply:
Good point, but I don't know how to avoid this code duplication.
The helper scripts run on each machine, and those scripts are not operating in terms of `BaseNode` or any other node.